lib/checks/shared/svg-non-empty-title.js

```diff
@@ -0,0 +1,4 @@
+const titleNode = virtualNode.children.find(({ props }) => {
+	return props.nodeName === 'title';
```

Comment:
I know this is a nitpick, but don't we always go `nodeName.toUpperCase`?

Reply:
The `VirtualNode` interface does that for us now https://github.com/dequelabs/axe-core/blob/develop/lib/core/base/virtual-node/virtual-node.js#L27-L36